templates/terraform/extra_schema_entry/backend_service.erb

```diff
@@ -0,0 +1,20 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2018 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+"region": {
+	Type:     schema.TypeString,
+	Optional: true,
+	ForceNew: true,
```

Comment:
Nope! `Removed` is effectively the same as being Computed and the field is unable to be set in config / interpolated on, so it will never cause a diff*. Also, this was the schema in the handwritten one afaik.

* Unless there's a bug in core, which is true of some nested fields.